pkg/kubectl/resource/visitor.go

```diff
@@ -474,14 +474,14 @@ func (v *StreamVisitor) Visit(fn VisitorFunc) error {
 			}
 			return err
 		}
-		ext.RawJSON = bytes.TrimSpace(ext.RawJSON)
-		if len(ext.RawJSON) == 0 || bytes.Equal(ext.RawJSON, []byte("null")) {
+		ext.Raw = bytes.TrimSpace(ext.Raw)
```

Comment:
This needs to know about the content encoding and type - I think we have a gap here for implementation (to be followed up later).  Add a `TODO: InfoForData needs to be able to handle objects in other encodings and schemas` and we should probably add this to the protobuf checklist on the protobuf issue.